Source/CombatExtended/CombatExtended/Things/Plant_Blazebulb.cs

```diff
@@ -21,14 +21,17 @@ public override void SpawnSetup(Map map, bool respawningAfterLoad)
         public override void TickLong()
         {
             base.TickLong();
-            float temperature = Position.GetTemperature(base.Map);
-            if (temperature > IgnitionTemp)
+            if (!Destroyed)
             {
```

Comment:
Nitpick: Prefer early returns

```suggestion
            if (Destroyed)
            {
              return;
            }
```